pkg/controller/servicebindingrequest/sbrcontroller.go

```diff
@@ -32,6 +32,9 @@ var (
 	// defaultPredicate default predicate functions
 	defaultPredicate = predicate.Funcs{
 		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectNew.GetObjectKind().GroupVersionKind().Kind == "secret" {
+				return false
```

Comment:
Should be `true` right?

Reply:
@sbose78 Yes. I have made a couple of more changes. Will push those changes in some time.